src/js/views/patients/sidebar/filters/filters-sidebar_views.js

```diff
@@ -35,7 +35,7 @@ const CustomFilterView = View.extend({
   className: 'flex flex-align-center u-margin--b-8',
   template: hbs`
     <h4 class="sidebar__label">{{ name }}</h4>
-    <div class="flex-grow" data-filter-button></div>
+    <div class="flex-grow u-text--overflow" data-filter-button></div>
```

Comment:
`overflow: hidden` is all that's needed. `.button-secondary` is what's actually applying the ellipsis to the text inside the button.

But otherwise we need to create a classname to do just `overflow: hidden`. Or inline styles I guess.